src/main/java/no/fishapp/app/chat/boundry/ChatResource.java

```diff
@@ -88,7 +76,7 @@ public Response startConversationRequest(
     @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
     public Response sendMessageRequest(
             @NotNull @PathParam("id") long conversationId,
-            @NotNull @HeaderParam("asdj") String messageBody
+            @NotNull @HeaderParam("body") String messageBody
```

Comment:
du planlegge vel ikke og ha msg body som et header param vel?

Reply:
Ikkje permanent, berre no for testing inntil vidare i appen.